app/commands/decidim/translation_addons/create_report.rb

```diff
@@ -13,7 +13,10 @@ def initialize(form, resource_instance, current_user)
       end
 
       def call
-        return broadcast(:invalid) if @resource_instance.blank? || @field.blank? || @locale.blank? || @current_user.blank? || @fix_suggestion.blank? || @reason.blank?
+        return broadcast(:invalid) if @resource_instance.blank? || @field.blank? || @locale.blank? || @current_user.blank? || @reason.blank?
```

Comment:
We have attr_reader, let's use it! 
```suggestion
        return broadcast(:invalid) unless [resource_instance, field,  locale, current_user, reason].all?
```